test/unit.test.js

```diff
@@ -51,6 +51,7 @@ for (const unitTest of fs.readdirSync(join(__dirname, 'unit'))) {
 
     const { fileList, reasons } = await nodeFileTrace([join(unitPath, inputFileName)], {
       base: `${__dirname}/../`,
+      processCwd: unitPath,
```

Comment:
I was expecting the base and processCwd to be the same here. Why is it different?

Reply:
I suppose base is the entire `test` directory where the process cwd is the `test/unit/<fixture>` so this seems correct 👍

Reply:
Let's also add `processCwd` to the `integration.test.js` too

Reply:
Ok, done.